nav2_bringup/launch/bt_navigator.xml

```diff
@@ -1,29 +1,16 @@
 <!--
-  This Behavior Tree first computes a path using the global planner (ComputePathToPose).
-  Then, it runs two sub-branches in parallel. The first sub-branch is a FollowPath
-  operation (the local planner). In parallel, there is a rate controlled execution of
-  FollowPath (the global planner). Each time a new path is computed, the path update
-  is sent to the local planner. The right branch, which is the rate controlled
-  ComputePathToPose, always returns RUNNING. Because the Parallel node uses a
-  threshold of 1, whenever the FollowPath returns SUCCESS or FAILURE, the parallel
-  node will return this result.
-
-  The goal (input to the global planner) and the resulting path (output of the global
-  planner and input to the local planner) are passed on the blackboard.
-
-  The rate at which the ComputePathToPose operation is invoked can be controlled with
-  the hz parameter to the RateController node.
+  This Behavior Tree replans the global path periodically at 1 Hz.
 -->
 
 <root main_tree_to_execute="MainTree">
   <BehaviorTree ID="MainTree">
     <Sequence name="root">
-      <RateController hz="1.0">
```

Comment:
If the GoalReached is not under the RateController, then it will be ticked at the same rate of FollowPath which is not necessary.  The GoalReached should be ticked at the same rate of ComputePathToPose.

Reply:
Yes, that's why I made the change. I want it to be ticked so that it will immediately detect that it is localized. Otherwise, the RateController will block the check of IsLocalized until the next time interval. This interval could be multiple seconds, depending on the hz setting. We've got it set at a second, which is then our worst case delay, but if someone upped that to 2 or 3 seconds, this could result in an unacceptable delay (FollowPath would have completed, but we're waiting on the RateController to let us through to IsLocalized).